### ISSUE TITLE
all images read as png

### ISSUE THREAD

**Issue by jacksonhart** [harryy2510/ngx-img#13]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?
> Windows 10 / macOS 10.14

### Versions
node 11.6
angular 7.2

### Issue
Both jpgs and png images are uploaded with 'data:image/png ... '

Is this issue specific to my environment?

**Comment by alijared** [comment]
Did you manage to resolve this?

**Comment by jacksonhart** [comment]
nope. we still seem to be able to use jpegs however. label still wrong though.
are you getting the same?
perhaps a regression as seems like this would have been picked up

**Comment by harryy2510** [comment]
Fixed in v10.19.3
